sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/src/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>An OpenTelemetry .NET distro that exports to Azure Monitor</Description>
+    <AssemblyTitle>AzureMonitor OpenTelemetry Distro</AssemblyTitle>
+    <Version>1.0.0-alpha.1</Version>
```

Comment:
Looks like CI is not happy with alpha version, may be changing to beta could help.